src/web/services/simple-modal.service.ts

```diff
@@ -44,6 +45,14 @@ export class SimpleModalService {
       Object.entries(simpleModalOptions).forEach(([key, value]: [string, string | boolean]) => {
         modalRef.componentInstance[key] = value;
       });
+      if (simpleModalOptions.onClosed) {
+        modalRef.closed.subscribe(() => {
+          simpleModalOptions.onClosed?.();
```

Comment:
Is `?` still needed? The existence of `onClosed` is already verified in the if block.

Reply:
![image](https://user-images.githubusercontent.com/75534016/170699028-85848d85-0f3f-4fe1-9085-1f2f62f93ec4.png)
This is the error that I got without the `?`, unless I suppress it.

Reply:
I see, method invocation is slightly stricter indeed

Reply:
@xyliew25, no need to change, but just a small tip: you can use a non null assertion to suppress this warning in situations where optional chaining is not suitable.

`simpleModalOptions.onClosed!()`

Reply:
@samuelfangjw thanks! I'll change it since I'm still editing.